node-scripts/spec/pipeline3/scripts.spec.js

```diff
@@ -188,10 +188,17 @@ describe("Scripts", function() {
       uuid: "1-22-333",
       scripts: [
         {
-          type: "shell",
+          type: "dockerExecBash",
           executionStage: "6",
           conditions: ["data"],
           value: "/a/script.sh"
+        },
+        {
+          type: "hostExecBash",
```

Comment:
Does this assume specifically Bash?

Reply:
Nothing really enforces bash.

Reply:
In that case, please rename to `hostExec`